output.R

```diff
@@ -161,36 +65,37 @@ choose_filename_prefix <- function(modules, title = "") {
 }
 
 if (exists("source_include")) {
-  comp <- FALSE
-} else if (!exists("comp")) {
-  comp <- choose_mode("Please choose the output mode")
+  comp <- "single"
```

Comment:
`choose_mode` was completely replaced by some pre- and postprocessing and a call to `chooseFromList`